modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/home/components/ProjectsList/ProjectList.test.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {faker} from '@faker-js/faker';
+import {render, screen} from '@testing-library/react';
+import ProjectList from '.';
+
+describe('render Project List component', () => {
```

Comment:
```suggestion
describe('ProjectList', () => {
```

No need to put this description inside describe, it makes more sense to me, applied to `it` or `test` suite

Reply:
Done, thank you!!